source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+may be used in the definition.
```

Comment:
"specifies that a template is being **declared** and that a *type-name* `T` may be used in the **definition**" simply doesn't make much sense to me, sorry. Similar to the argument for "may" here, at this point in time we don't know if the prefix is being followed by a definition or a (non-defining) declaration yet, so I still think "declaration" makes more sense. But at least the two highlighted words should match.